.babelrc

```diff
@@ -1,20 +1,23 @@
 {
-  "stage": 0,
+  "presets": ["es2015", "react", "stage-0"],
   "env": {
     "browser": {
       "plugins": ["react-transform"],
```

Comment:
the `extra` key has been removed from config obj in babel 6. when options to plugin array in `env.browser`, wrap it [in an array and provide the options object](https://babeljs.io/docs/plugins/#options) as the second item... ie:

```javascript
"plugins": [
  ["react-transform", {
    "transforms": [{
      "transform": "react-transform-hmr",
      "imports": ["react"],
      "locals": ["module"]
    }, {
      "transform": "react-transform-catch-errors",
      "imports": [
        "react",
        "redbox-react"
      ]
    }]
  }]
]
```